src/main/java/com/onarandombox/MultiversePortals/listeners/MVPPlayerListener.java

```diff
@@ -161,12 +162,14 @@ public void playerInteract(PlayerInteractEvent event) {
 
         Material itemType = plugin.getWandMaterial();
         // If we Found WorldEdit, return, we're not needed here.
-        // If the item is not the Wand we've stetup we're not needed either
+        // If the item is not the Wand we've setup we're not needed either
```

Comment:
extremely minor nitpick - i believe it's actually `set up` unless we're talking about the noun `setup`.